src/cli.ts

```diff
@@ -0,0 +1,294 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+'use strict';
+
+import * as commandLineArgs from 'command-line-args';
+import * as fs from 'fs';
+import {Runner, RunnerOptions} from './runner';
+
+export interface CliOptions {
+  'config-file'?: string;
+  'exclude-repo'?: string[];
+  'github-token'?: string;
+  'fresh'?: boolean;
+  'help'?: boolean;
+  'latest-release'?: boolean;
+  'repo'?: string[];
+  'skip-test'?: string[];
+  'test'?: string[];
+  'verbose'?: boolean;
+  'wct-flags'?: string[];
+  'workspace-dir'?: string;
+}
+
+export interface ConfigFileOptions {
+  'exclude-repo'?: string[];
+  'github-token'?: string;
+  'fresh'?: boolean;
+  'latest-release'?: boolean;
+  'repo'?: string[];
+  'skip-test'?: string[];
+  'test'?: string[];
+  'verbose'?: boolean;
+  'wct-flags'?: string[];
+  'workspace-dir'?: string;
+}
+
+export function getCommandLineOptions(): CliOptions {
+  return <CliOptions>commandLineArgs(cliOptionDefinitions);
+}
+
+export function commandLineArgsUsageOptsToCommandLineUsageOptionList(
```

Comment:
This function was unnecessary and also dumb.  I typed it out REALLY fast at the end of the day not realizing the data structure I was producing was already compatible with the command-line-usage function package as-is.  So I have removed.  Fix in a1c00e053969ec5056efe4a48b343a0ccd0623c0